### PR TITLE
Refactors Video.js adding a Video class

### PR DESCRIPTION
**Pull request by jaysaleh**
This pull request adds a `Video` class.

#259 Needs to be addressed as a result of this.

### CODE REVIEW

File: src/main/webapp/view/video/video.js [hunk 1]
```diff
@@ -16,40 +16,42 @@ import {startVideoSyncTimer} from '../../synchronizer.js';
 
 const SCRIPT = 'script';
 
-window.onYouTubeIframeAPIReady = onYouTubeIframeAPIReady;
+// TODO: Add function that returns current video time #259.
+export default class Video {
```
Comment (amuamushu) [inline src/main/webapp/view/video/video.js:20]: can you add a class header?

File: src/main/webapp/view/video/video.js [hunk 2]
```diff
@@ -16,40 +16,43 @@ import {startVideoSyncTimer} from '../../synchronizer.js';
...
-    events: {
-      onReady: onPlayerReady,
-    },
-  });
-}
+  /**
+   * Creates a YouTube Video iFrame that plays lecture video after
+   * the API calls it. This is a required callback from the API.
+   */
+  // TODO: Change height and width.
```
Comment (billie-cat) [inline src/main/webapp/view/video/video.js:35]: nit: I would re-word this TODO to be more specific. 

Support dynamic video height and width.

File: src/main/webapp/view/video/video.js [hunk 3]
```diff
@@ -16,40 +16,43 @@ import {startVideoSyncTimer} from '../../synchronizer.js';
...
+    startVideoSyncTimer();
+  }
 
-/** Seeks video to `currentTime`. */
-export function seekVideo(currentTime) {
-  // TODO: Removed and implement.
-  console.log('SEEKING VIDEO TO: ' + currentTime);
+  /** Seeks video to `currentTime`. */
+  seekVideo(currentTime) {
+    // TODO: Removed and implement.
```
Comment (billie-cat) [inline src/main/webapp/view/video/video.js:55]: I know Nathan also made changes to this code. Who is submitting first?

Comment (jaysaleh) [reply src/main/webapp/view/video/video.js:55]: Coordinating with @n-wach

**Pull request merged**